tool.go

```diff
@@ -560,7 +560,7 @@ type serveCommandFileSystem struct {
 }
 
 func (fs serveCommandFileSystem) Open(requestName string) (http.File, error) {
-	name := path.Join(fs.serveRoot, requestName)
+	name := path.Join(fs.serveRoot, requestName[1:]) //requestName[0] == '/'
```

Comment:
In Go, the convention for user-facing comments is to have a single space between the `//` (or `/*`) and the text. So this comment should be:

```Go
// requestName[0] == '/'
```

The only times not to have a space is when it's a temporarily commented out development code (usually, that code is not committed, it's a temporary state during development) and for commands meant for non-compiler programs. For example, `//go:generate ...`.